cvat-ui/src/components/task-page/job-list.tsx

```diff
@@ -53,6 +53,9 @@ function setUpJobsList(jobs: Job[], query: JobsQuery): Job[] {
         result = result.filter((job, index) => jsonLogic.apply(filter, converted[index]));
     }
 
+    // primarily only normal jobs should be shown
```

Comment:
If we use `parentJobId` parameter, that means not only normal jobs are shown, but GT jobs are also shown

Reply:
Sorry for not writing the comment properly. Primarily only normal jobs and GT jobs would be shown as the consensus job will be collapsed inside the corresponding normal job.